docs/en/04_Changelogs/4.0.0.md

```diff
@@ -1320,12 +1383,30 @@ handle field-level and form-level messages. This has the following properties:
   for all DataObject subclasses, rather than just the basename without namespace.
 * i18n key for locale-respective pluralisation rules added as '.PLURALS'. These can be configured
   within yaml in array format as per [ruby i18n pluralization rules](http://guides.rubyonrails.org/i18n.html#pluralization).
+* `i18n.all_locales` config moved to `Locales.locales`
```

Comment:
All of these should mention the correct namespace, otherwise it's harder to fix via copy/paste.